### DISCUSSION TITLE
Which ESP8266 NodeMCU

### DISCUSSION THREAD

**Discussion by SigmaPic** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#315] [Q&A] [answered]
Hello,

I would like to buy the rigth board.

The guide says : ESP8266 NodeMCU 1.0 (12E) V3

When looking on internet you can find different types:
- CP2102
- CH340

Which is the good one ? The CP2102 ?

For example : [eBay](https://www.ebay.fr/itm/133492399084?_trkparms=amclksrc%3DITM%26aid%3D1110002%26algo%3DSPLICE.SOI%26ao%3D1%26asc%3D20201210111451%26meid%3Df51a8c85456b4e41b2b84ee9d350f7da%26pid%3D101196%26rk%3D1%26rkt%3D12%26sd%3D133464799049%26itm%3D133492399084%26pmt%3D1%26noa%3D0%26pg%3D2047675%26algv%3DPromotedSellersOtherItemsV2WithMLRv3%26brand%3D-+Sans+marque%2FG%E9n%E9rique+-&_trksid=p2047675.c101196.m2219&amdata=cksum%3A133492399084f51a8c85456b4e41b2b84ee9d350f7da%7Cenc%3AAQAHAAABELB59iZaq6L1SxCGLGoiBqHdpPz8nqKyKuGL1QIa0H%252B4hCIdtLYUbubGdYzDDIGnd5h4crIkuL7%252FKKCx9vyglxYZLZZ%252B%252BCVu3YgpqOnvc24Qz%252FXRloKhR9Ojj1P%252BnNNZrzJEea96ZC4znPTIVksuKuwQuJIjE8YTI4fbWquUuo%252F3cQ32STv7TL1uLKO%252FoOODWrapB2b6hjiO%252BC1ofMqAvL8jShiKajhjHPzORJF6g1M7QEEZppG8oUpb3l61rFLP8pHMT70pM1v5FPasRh2aktYIwWXZyp9Nr0Lj42oRIprLxqrg09WclRud%252FdjW5bz8tyCGVbwzxu%252F57vX6nWdbOWAX%252Bo6ojzxXkU2GPjj%252BirPC%7Campid%3APL_CLK%7Cclp%3A2047675)

**Comment by visualapproach** [comment, marked as answer]
Doesn't matter

**Reply by SigmaPic** [reply]
Ok, only different serial converter
